integration_test/src/lib.rs

```diff
@@ -47,6 +47,18 @@ pub trait NodeExt {
 
     /// Generates [`NBLOCKS`] to an address controlled by the loaded wallet.
     fn fund_wallet(&self);
+
+    /// Mine a block.
+    ///
+    /// Should send mining reward to a new address for the loaded wallet.
+    fn mine_a_block(&self);
+
+    /// Create a transaction and mine it.
+    ///
+    /// # Returns
+    ///
+    /// The receive address and the transaction. 
```

Comment:
In 3e17653dbfa9acc451b0ccc4d8a8713b1aac154e:

Trailing whitespace.

Reply:
Were are the git hooks!!!